module.flow.js

```diff
@@ -411,9 +411,11 @@ declare module '@solana/web3.js' {
         Transaction | TransactionInstruction | TransactionInstructionCtorFields,
       >
     ): Transaction;
+    signData: Buffer;
```

Comment:
How about `serializeMessage`? "Data" is not very clear as to what's being signed

Reply:
👍 starting to move the JS SDK towards exposing `Message` in the same way as the Rust SDK does would be nice

Reply:
WFM!